src/content/docs/guides/python/python-basics.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Python Basics
+description: Learning about the basics of Python
+sidebar:
+    order: 3
+---
+
+We're going to start off with a couple very basic things you can do in python.
+
+## Printing
+
+Printing lets you output something like a word or some numbers and even more. The `print()` function will display the given value on screen.
+
+```python
+print("Hello, World!")
```

Comment:
Maybe add a section on using print to do math? Could also be combined with variables as it's normally a fun one to play around with tos ee if you can break python.